### PR TITLE
Benchmark related maintenance improvements

### PR DESCRIPTION
**Pull request by medikoo**
- Improve documentation
- Ensure to gitignore go build results

### CODE REVIEW

File: node/test/benchmark/README.md [hunk 1]
```diff
@@ -105,18 +105,44 @@ Max memory used as reported by AWS (with `maxMemoryUsed` metric provided in invo
...
+
+```
+python3 -m venv .venv
+source .venv/bin/activate
+python3 -m pip install serverless-aws-lambda-sdk --target=python/packages/aws-lambda-sdk/dist
+```
+
+#### 1.3 Go
+
+Ensure Go being installed in the environment
```
Comment (Danwakeem) [inline node/test/benchmark/README.md]:
```suggestion
Ensure Go is installed in the environment
```

**Review by Danwakeem — DISMISSED** [review]
Looking good just one small typo 👍

File: node/test/benchmark/README.md (hunk 1 above)
Comment (medikoo) [reply node/test/benchmark/README.md]: Thanks, updated

**Review by Danwakeem — APPROVED** [review]
:shipit:

**Pull request merged**